.ci/get-boost.sh

```diff
@@ -42,6 +42,7 @@ git submodule --quiet update --init $GIT_SUBMODULE_OPTS \
     libs/filesystem \
     libs/integer \
     libs/iterator \
+    libs/math \
```

Comment:
This seems like remains from previous round of reviews in https://github.com/BoostGSoC19/gil-olzhas/pull/1

Reply:
Indeed, I guess I did reset incorrectly, should've double checked ... Was it previously at transitive requirements?

Reply:
Yes, it is transitive

https://github.com/boostorg/gil/blob/0fe841f2bf3f8ba25891bc0b3d1fac25ac5cd050/.ci/get-boost.sh#L52-L69